### ISSUE TITLE
Linked inline image has no image toolbar when selected

### ISSUE THREAD

**Issue by oleq** [ckeditor/ckeditor5#9522]
## 📝 Provide detailed reproduction steps (if any)

![](https://user-images.githubusercontent.com/1099479/115390948-2b3b0800-a1df-11eb-926e-e239b6a0a92e.gif)

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by oleq** [comment]
Looks like this helper got lost when resolving conflicts

```
getSelectedImageWidget( selection ) {
		const viewElement = selection.getSelectedElement();

		if ( viewElement ) {
			if ( this.isImageWidget( viewElement ) ) {
				return viewElement;
			}

			// If a selected inline image widget is the only child of a link, the selection will encompass
			// that link. But this still counts as a selected image widget. This is what it looks like:
			// [<a href="..."><span class="image-inline ck-widget ck-widget_selected"><img ... /></span></a>]
			if ( viewElement.is( 'element', 'a' ) && viewElement.childCount === 1 ) {
				const firstChild = viewElement.getChild( 0 );

				if ( firstChild.is( 'element' ) && this.isImageWidget( firstChild ) ) {
					return firstChild;
				}
			}
		}
```

and it was used here https://github.com/ckeditor/ckeditor5/blame/0cb8e4c8cb6ffaac3359831e79688f1e627b3fc4/packages/ckeditor5-image/src/imagetoolbar.js#L55

**Comment by oleq** [comment]
Closed in 6089549.
